tutorials/exclusion_mask.ipynb

```diff
@@ -0,0 +1,348 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating exclusion masks\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "### Prerequisites\n",
+    "\n",
+    "- Understanding of basic analyses in 1D or 3D.\n",
+    "- Usage of `regions` and catalogs. \n",
+    "\n",
+    "### Context\n",
+    "\n",
+    "Background templates stored in the DL3 IRF are often not reliable enough to be used without some corrections. A set of common techniques to perform background or normalisation from the data is implemented in gammapy: reflected regions for 1D spectrum analysis, field-of-view (FoV) background or ring background for 2D and 3D analyses.\n",
+    "\n",
+    "To avoid contamination of the background estimate from gamma-ray bright regions these methods require to exclude those regions from the data used for the estimation. To do so, we use exclusion masks. They are maps containing boolean values where excluded pixels are stored as False.  \n",
+    "\n",
+    "**Objective: Build an exclusion mask around the Crab nebula excluding gamma-ray sources in the region.**\n",
+    "\n",
+    "### Proposed approach\n",
+    "\n",
+    "Here we have to build a `Map` object, we must first define its geometry and then we can determine which pixels to exclude.\n",
+    "\n",
+    "We can rely on known sources positions and properties to build a list of regions (here `~regions.SkyRegions`) enclosing most of the signal that our detector would see from these objects. We show below how to build this list manually or from an existing catalog. \n",
+    "\n",
+    "Finally, we show how to build the mask from a `MapDataset`, finding pixels which contain statistically significant signal. To do so, we use the `LiMaMapEstimator`\n",
```

Comment:
Maybe you can add a line saying that the different sections show 3 different, independent methods of building masks.
Also, if users want, they can club different methods using simple map arithmetics, eg: `exclusion = exclusion1 * exclusion2`

Reply:
Good point. Added to the notebook.